pkg/engine/plugins.go

```diff
@@ -46,27 +46,23 @@ const (
 // PluginSet represents a set of plugins.
 type PluginSet map[string]workspace.PluginSpec
 
+// NewPluginSet creates a new PluginSet from the specified PluginSpecs.
+func NewPluginSet(plugins ...workspace.PluginSpec) PluginSet {
+	var s PluginSet = make(map[string]workspace.PluginSpec, len(plugins))
+	for _, p := range plugins {
+		s.Add(p)
+	}
+	return s
+}
+
 // Add adds a plugin to this plugin set.
 func (p PluginSet) Add(plug workspace.PluginSpec) {
 	p[plug.String()] = plug
 }
 
-// Union returns the union of this pluginSet with another pluginSet.
-func (p PluginSet) Union(other PluginSet) PluginSet {
-	newSet := NewPluginSet()
-	for _, value := range p {
-		newSet.Add(value)
-	}
-	for _, value := range other {
-		newSet.Add(value)
-	}
-	return newSet
-}
```

Comment:
This method isn't needed any more (it's moved to `PackageSet`), so it has been deleted from `PluginSet` for now.